coderd/database/dbauthz/dbauthz.go

```diff
@@ -253,6 +253,7 @@ var (
 					rbac.ResourceDeploymentConfig.Type:       {policy.ActionCreate, policy.ActionUpdate, policy.ActionDelete},
 					rbac.ResourceNotificationPreference.Type: {policy.ActionCreate, policy.ActionUpdate, policy.ActionDelete},
 					rbac.ResourceNotificationTemplate.Type:   {policy.ActionCreate, policy.ActionUpdate, policy.ActionDelete},
+					rbac.ResourceTailnetCoordinator.Type:     {policy.ActionCreate, policy.ActionUpdate, policy.ActionDelete},
```

Comment:
the "system" identity is this gnarly kitchen sink we have.  We shouldn't be adding to it because the PGCoordinator already has an identity with correct permissions.

Reply:
I thought I removed this honestly, will fix